### PR TITLE
Regenerate azure-search-documents on New SHA

### PR DESCRIPTION
**Pull request by alzimmermsft**
# Description

This PR updates Search to generate based on a new SHA for the Swaggers.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](../../CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by pallavit — APPROVED** [review]
:shipit:

File: sdk/search/azure-search-documents/swagger/README.md [hunk 1]
```diff
@@ -72,10 +72,10 @@ These settings apply only when `--tag=package-2021-04-30-Preview-searchservice`
...
...DataContainer,SearchIndexerDataIdentity,SearchIndexerDataNoneIdentity,SearchIndexerDataSourceType,SearchIndexerDataUserAssignedIdentity,SearchIndexerError,SearchIndexerKnowledgeStore,SearchIndexerKnowledgeStoreBlobProjectionSelector,SearchIndexerKnowledgeStoreFileProjectionSelector,SearchIndexerKnowledgeStoreObjectProjectionSelector,SearchIndexerKnowledgeStoreProjection,SearchIndexerKnowledgeStoreProjectionSelector,SearchIndexerKnowledgeStoreTableProjectionSelector,SearchIndexerLimits,SearchIndexerSkill,SearchIndexerSkillset,SearchIndexerStatus,SearchIndexerWarning,SearchIndexStatistics,SearchResourceEncryptionKey,SearchServiceCounters,SearchServiceLimits,SearchServiceStatistics,SearchSuggester,SemanticConfiguration,SemanticField,SemanticSettings,SentimentSkill,SentimentSkillLanguage,ShaperSkill,SimilarityAlgorithm,SnowballTokenFilterLanguage,SoftDeleteColumnDeletionDetectionPolicy,SplitSkill,SplitSkillLanguage,SqlIntegratedChangeTrackingPolicy,StemmerTokenFilterLanguage,StopAnalyzer,StopwordsList,SynonymMap,TagScoringFunction,TagScoringParameters,TextSplitMode,TextTranslationSkill,TextTranslationSkillLanguage,TextWeights,TokenCharacterKind,TokenFilterName,VisualFeature,WebApiSkill
```
Comment (pallavit) [inline sdk/search/azure-search-documents/swagger/README.md:78]: I wish we had a way to inverse this. I.e. what to only put in implementation for scenarios like this where the list is huge!

Comment (alzimmermsft) [reply sdk/search/azure-search-documents/swagger/README.md:78]: That should be possible by inverting the `models-subpackage` and the `custom-types-subpackage` but this way is better as we won't generate into the public API directly, and it reduces the chance that we accidentally release an API.

Comment (srnagar) [reply sdk/search/azure-search-documents/swagger/README.md:78]: Search has a long list of models and inverting would also have a large number of files to list here I guess. Anyway, the alternative is what Alan is suggesting.

**Pull request merged**